docs/schema/guidance/languages.rst

```diff
@@ -1,9 +1,29 @@
-Languages and transliteration
-=============================
+Representing alternative languages
+==================================
 
-There may be times when beneficial ownership information needs to be provided in different languages.
+Beneficial ownership information may need to be provided in different languages or scripts. BODS provides options for representing translated or transliterated information. 
 
-(1) For people's names, use an entry in the ``names`` array with a ``type`` of 'translation'
-(2) For entity names, use an entry in the ``alternativeNames`` array
+To represent:
+
+* people’s names, use an entry in the names array with ``name.type`` ‘translation’ or ‘transliteration’ 
+
+* entity names, use an entry in the ``alternativeNames`` array
+
+* entity types use ``entitySubtype.localTerm``
+
+* interest types use ``interest.details``
+
+For other fields or entire records, use an annotation. 
+
+In the annotation:
+
+* ``motivation`` MUST be ‘transformation’
+
+* ``description`` SHOULD explain the type of transformation (e.g. ‘translation’) 
+
+* ``transformedContent`` MUST contain the transformed content
+
+Translation converts text from one language to another, preserving the meaning of the text. For example the Russian word “Здравствуйте” can be translated to “hello” in English. 
```

Comment:
Maybe we should just delete these last two paras and leave people to look up terms if they don't know them.

If not, we somehow need to refer to these two paras from the second sentence of the page, imo.

Reply:
Yeah agreed - originally I had these at the top of the page but didn't want to prioritise over the normative content. This information will also be held in the codelist for nameType.